include/uapi/sound/sof/tokens.h

```diff
@@ -88,6 +88,7 @@
 #define SOF_TKN_COMP_CPC			406
 #define SOF_TKN_COMP_IS_PAGES			409
 #define SOF_TKN_COMP_NUM_AUDIO_FORMATS		410
+#define SOF_TKN_COMP_OUTPUT_CNT		411
```

Comment:
This will be conflicted with https://github.com/thesofproject/linux/pull/3757 @aiChaoSONG FYI

Reply:
Got it. Thanks. Will use another value.